### ISSUE TITLE
Option to ignore files outside of p4 root

### ISSUE THREAD

**Issue by tivac** [stef-levesque/vscode-perforce#9]
I use vscode for editing files both within a p4 root and external to it. Because I'm lazy I have it configured like this:

```js
// Automatically open a file for edit when saved
"perforce.editOnFileSave": true,

// Automatically open a file for edit when Modified
"perforce.editOnFileModified": true,

// Automatically Add a file to depot when Created
"perforce.addOnFileCreate": true,

// Automatically delete a file from depot when deleted
"perforce.deleteOnFileDelete": true,
```

which is awesome while under a p4 root, and super-annoying outside of it. Would an option to ignore errors if not under a p4 root be possible?

I experimented with setting those all to `false` by default and just overriding in a `.vscode/settings.json` file, but I would need to add that all over my projects for that to work since vscode doesn't seem to walk up the directory structure to find settings.

**Comment by stef-levesque** [comment]
Should be fixed now, try updating to vscode-perforce@0.1.4

**Comment by tivac** [comment]
Awesome, thanks! I'll poke at this tomorrow when I'm back at my P4-using machine :+1:
